lib/shared/index.ts

```diff
@@ -100,6 +103,88 @@ export class Shared extends Construct {
         service: ec2.InterfaceVpcEndpointAwsService.SAGEMAKER_RUNTIME,
         open: true,
       });
+      
+    }
+    
+    if (props.config.privateWebsite) {
+      // Create VPC Endpoint for AppSync
+      vpc.addInterfaceEndpoint("AppSyncEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.APP_SYNC,
+      });
+      
+      // Create VPC Endpoint for Lambda
+      vpc.addInterfaceEndpoint("LambdaEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.LAMBDA,
+      });
+      
+      // Create VPC Endpoint for Kendra
+      vpc.addInterfaceEndpoint("KendraEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.KENDRA,
+      });
+      
+      // Create VPC Endpoint for SNS
+      vpc.addInterfaceEndpoint("SNSEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.SNS,
+      });
+      
+      // Create VPC Endpoint for Step Functions
+      vpc.addInterfaceEndpoint("StepFunctionsEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.STEP_FUNCTIONS,
+      });
+     
+      // Create VPC Endpoint for SSM
+      vpc.addInterfaceEndpoint("SSMEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.SSM,
+      });
+      
+      // Create VPC Endpoint for RDS
+      vpc.addInterfaceEndpoint("RDSEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.RDS,
+      });
+      
+      // Create VPC Endpoint for RDS Data
+      vpc.addInterfaceEndpoint("RDSDataEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.RDS_DATA,
+      });
+      
+      // Create VPC Endpoint for ECR
+      vpc.addInterfaceEndpoint("ECREndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.ECR,
+      });
+      
+      // Create VPC Endpoint for ECR Docker
+      vpc.addInterfaceEndpoint("ECRDockerEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.ECR_DOCKER,
+      });
+      
+      // Create VPC Endpoint for ECS
+      vpc.addInterfaceEndpoint("ECSEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.ECS,
+      });
+      
+      // Create VPC Endpoint for Batch
+      vpc.addInterfaceEndpoint("BatchEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.BATCH,
+      });
+      
+      // Create VPC Endpoint for EC2
+      vpc.addInterfaceEndpoint("EC2Endpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.EC2,
+      });
+      
+      // Create VPC Endpoint for KMS
+      vpc.addInterfaceEndpoint("KMSEndpoint", {
+          service: ec2.InterfaceVpcEndpointAwsService.KMS,
+      });
+      
+      // // Create VPC Endpoint for Bedrock 
```

Comment:
Why is this commented out?

Reply:
Added Bedrock VPC Endpoint for supported regions: https://github.com/aws-samples/aws-genai-llm-chatbot/pull/310/commits/a67629d96c0d02eef93b223327fa4011c0316e14